app/components/user_list/__snapshots__/index.test.tsx.snap

```diff
@@ -517,6 +517,86 @@ exports[`components/channel_list_row should show results and tutorial 1`] = `
           </View>
         </View>
       </View>
+      <Modal
```

Comment:
Why did your changes create such a big change in the snapshot?

Reply:
cause this is showing the tutorial as a result of the third test `should show results and tutorial`